python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -169,6 +170,32 @@ class EtcsBrakeParams(BaseModel, extra="forbid"):
     t_be: float = Field(ge=0, description="T_be: safe brake build up time in s")
 
 
+class SimpleSignalingSystem(BaseModel, extra="forbid"):
+    signaling_system_type: Literal["BAL", "BAPR", "TVM300", "TVM430"]
+
+    @model_serializer
+    def ser(self) -> Mapping[str, Any]:
+        return {"type": self.signaling_system_type}
+
+
+class EtcsSignalingSystem(BaseModel, extra="forbid"):
+    signaling_system_type: Literal["ETCS_LEVEL2"]
```

Comment:
To avoid passing it as there is only one possible value:
```suggestion
    signaling_system_type: Literal["ETCS_LEVEL2"] = "ETCS_LEVEL2"
```
(param name could change, but still)

Reply:
done in https://github.com/OpenRailAssociation/osrd/pull/13435/commits/4d0efb625492f081542d410931521701bc5cddc4